pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -41,8 +41,7 @@ type EventDispatcher interface {
 
 	// PostStopPod is invoked after all of a pod's containers have permanently
 	// stopped running, but before the pod sandbox is destroyed.
-	PostStopPod(pod *v1.Pod, cgroupPath string) (*lifecycle.EventReply, error)
-
+	PostStopPod(cgroupPath string) error
```

Comment:
@ConnorDoyle As you said, sending this pod spec is helpful, but gather this information while destroying pod is not so easy(but imo doable). Maybe I'm too wary, but I'm afraid that this approach could block as in the future until pushing to core repository. Anyway we need to bypass some unique information about pod to isolator - w/o it there won't be possible to reclaim reservations from resources.